bin/pytorch_inference/connect_pipe.py

```diff
@@ -0,0 +1,103 @@
+import os
+import stat
+import time
+
+
+
+# The pipes are created by the C++ app
+# The names must match those passed to app
+restore_pipe_name = 'restore_pipe'
+input_pipe_name = 'input_pipe'
+output_pipe_name = 'output_pipe'
+
+
+source_file_name = '/Users/davidkyle/source/ml-search/projects/universal/torchscript/dbmdz-ner/conll03_traced_ner.pt'
+file_size = 1330816933
```

Comment:
I assume the first pass is just hardcoding everything to get it all working or did you want to parameterize this now?

Reply:
I updated the script removing the hardcoding. parameterised